rclpy/package.xml

```diff
@@ -34,6 +34,7 @@
   <depend>rmw_implementation</depend>
   <depend>rosidl_runtime_c</depend>
   <depend>unique_identifier_msgs</depend>
+  <depend>python3-yaml</depend>
```

Comment:
This should be an `exec_depend`, since we only require this at runtime, not build-time.  Also, please put it in alphabetical order with the rest of the `exec_depend` below.

Reply:
fixed but not sure why build failed